ontology/vo.md

```diff
@@ -25,6 +25,29 @@ publications:
   title: Mining of vaccine-associated IFN-γ gene interaction networks using the Vaccine Ontology
 repository: https://github.com/vaccineontology/VO
 tracker: https://github.com/vaccineontology/VO/issues
+usages:
+- description: VIOLIN uses VO to standardize vaccine information
+  examples:
+  - description: VIOLIN using VO grouped all SARS-CoV-2 vaccines
+    url: https://violinet.org/canvaxkb/vaccine_detail.php?c_vaccine_id=5339
+  - description: A specific vaccine ‘Allogeneic Tumor Cell Vaccine’ curated in VO for VIOLIN vaccine record
+    url: https://violinet.org/vaxquery/query_detail.php?c_pathogen_id=321#vaccine_5878
```

Comment:
There doesn't seem to be anything rendered as results for this query URL.
Is there some issues on the backend that is affecting the search and retrieval?

Reply:
Due to the system update, the link did not work. The issue has been resolved. The URL is working now.

Reply:
Ah yes. The link works now. Thank you for clarifying!